tests/foreman/api/test_discoveredhost.py

```diff
@@ -313,7 +313,7 @@ def test_positive_auto_provision_all(
         :expectedresults: All discovered hosts should be auto-provisioned
             successfully
 
-        :CaseAutomation: NotAutomated
+        :CaseAutomation: Automated
```

Comment:
@adarshdubey-star can you confirm this change? I see you were the one who automated it in [PR#12815](https://github.com/SatelliteQE/robottelo/pull/12815/files#diff-b90f667b541a4048392f93ebd6c5636837f8cb29b4bdf6c1f6ea31d7b2b8b945R307).